_includes/alert.html

```diff
@@ -5,28 +5,14 @@
 {% assign body = include.body | markdownify | remove: '<p>' | remove: '</p>' | strip %}
 
 <div
-    class="alert {% if header != blank and header != empty and header != nil %}alert-complex{% endif %} {{ type_class }}">
-    {% if header != blank and header != empty and header != nil %}
-    <header class="alert-header">
-        {% if icon != blank and icon != empty and icon != nil %}
-        <i class="material-icons alert-icon">{{ icon }}</i>
-        {% endif %}
-
-        {% if header != blank and header != empty and header != nil %}
-        <h3>{{ header }}</h3>
-        {% endif %}
-    </header>
-    {% else %}
+    class="alert {{ type_class }}">
     {% if icon != blank and icon != empty and icon != nil %}
     <i class="material-icons alert-icon mr-3">{{ icon }}</i>
     {% endif %}
-    {% endif %}
 
     {% if body != blank and body != empty and body != nil %}
     {% if header != blank and header != empty and header != nil %}
-    <div class="alert-body">
-        <p>{{ body }}</p>
-    </div>
+        <p><b>{{ header }}:</b> {{ body }}</p>
```

Comment:
I'm not a fan of using the `<b>` tag, but I see that's what's being used in the Design Guide, so this is something we should elsewhere @PhungNg.